### ISSUE TITLE
Batch normalization is not trained.

### ISSUE THREAD

**Issue by mensaochun** [DetectionTeamUCAS/Faster-RCNN_Tensorflow#10]
Thank you for sharing your code, I have used it in my own project.
I have another question. you do note use BN to train resnet, since batch size is too small(The code only supports batch size 1), it works well when training voc data。I think voc data and imageNet are all natural scene data, so the BN parameters can be transfered. But, considering the image in industrial application, the image differ much with natural image,  is it appropriate to use fixed BN parameters, should I open the BN training switch?
Thank you again.

**Comment by yangxue0827** [comment]
@mensaochun Your analysis makes sense, but we have not tried it. You can try it out and share the conclusions here.

**Comment by yangJirui** [comment]
You should train BN with large batch_size, but most GPUs don't have enough memory to allow us train Detection-Code with large batch_size. So If you do want to train the BN parameters, i have some advices as following:
1. you should train the detection-code with multi-GPU. Unfortunately, our code hasn't supported training with multi-GPU.
2. Sync Batch-Norm is necessary. Because most of multi-GPU training only compute the mean and var in single GPU. However, Sync is not supported by tensorflow, so it's not a good choice to use tensorflow to coding. But MxNet supported Sync BN recently, which is so convenient。
3. Group BN is another choice, but we don't have try it. 
@mensaochun
